docs/development/use-bundled-plugins.md

```diff
@@ -0,0 +1,63 @@
+# Using bundled plugins in local development
+
+## You want a quick way to add the security plugin to your local env
+
+### 1. Start an instance of OpenSearch that already includes the security plugin
+
+By default, we generally develop against a "minimal" version of OpenSearch, with no external plugins installed (such as when [running `yarn opensearch snapshot`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#run-opensearch)). This is great to make sure we understand the basic behavior of the core app. But that's not how most users install or run OpenSearch - they generally use all the bundled "default" plugins, of which [`security`](https://github.com/opensearch-project/security) is probably the most important.
```

Comment:
We should also update the snapshot command to pull the bundled version of opensearch if we pass a flag like `--bundled`. That will make many of the next steps unnecessary :)